console/src/components/Common/utils/v1QueryUtils.ts

```diff
@@ -195,6 +195,35 @@ export const getInsertDownQuery = (
   return getRunSqlQuery(sql);
 };
 
+export const getEditRowQuery = (
+  tableDef: TableDefinition,
+  item: Record<string, PGReturnValueType>,
+  columns: BaseTableColumn[],
+  pkClause: Record<string, PGReturnValueType>
+) => {
+  const columnValues = Object.keys(item).map(key => `"${key}"`);
+
+  const values = Object.values(item).map((value, valIndex) => {
+    const colVal =
+      columns.find(col => col.column_name === Object.keys(item)[valIndex]) ??
+      columns[valIndex];
+    return convertPGValue(value, colVal);
+  });
+
+  const updatedValues = columnValues
+    .map((value, valIndex) => {
+      return `${value} = ${values[valIndex]}`;
+    })
+    .join(', ');
+
+  const clauses = Object.keys(pkClause).map(pk =>
```

Comment:
this might break some cases. please be sure to test and check the working of this method upon making all of these changes.